src/hct_mis_api/apps/program/fixtures.py

```diff
@@ -140,3 +140,74 @@ def get_program_with_dct_type_and_name(
         **kwargs,
     )
     return program
+
+
+def generate_beneficiary_groups() -> None:
+    BeneficiaryGroupFactory(
+        **{
+            "name": "Household",
+            "group_label": "Household",
+            "group_label_plural": "Households",
+            "member_label": "Individual",
+            "member_label_plural": "Individuals",
+            "master_detail": True,
+        }
+    )
+    BeneficiaryGroupFactory(
+        **{
+            "name": "Social Workers",
+            "group_label": "Household",
+            "group_label_plural": "Households",
+            "member_label": "Individual",
+            "member_label_plural": "Individuals",
+            "master_detail": False,
+        }
+    )
+
+
+def generate_people_program() -> None:
+    from hct_mis_api.apps.household.fixtures import (
+        create_household,
+        create_individual_document,
+    )
+    from hct_mis_api.apps.household.models import HOST, SEEING
+
+    ba = BusinessArea.objects.get(name="Afghanistan")
```

Comment:
this can become a fixture

https://docs.pytest.org/en/6.2.x/fixture.html

@pytest.fixture
def afghanistan():
    return BusinessAreaFactory(name="Afghanistan")

Reply:
this one as well used only in `initdemo.py`